.github/workflows/pull-request.yml

```
@@ -272,35 +272,71 @@
           path: "./coverage/vitest/lcov.info"
           min_coverage: 0
+        
+      - name: Validate GraphQL schema
+        run: |
+          # Check if any .graphql files exist
+          if ! find src -name "*.graphql" -type f | grep -q .; then
+            echo "Warning: No .graphql files found in src directory"
+            exit 0
+          fi
+          npx graphql-inspector validate schema.graphql "src/**/*.graphql"
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)